src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -191,18 +212,20 @@ private async Task UpdateInstalledVersionsAsync(CancellationToken cancellationTo
         /// This method is called from several methods that are called from properties and LINQ queries
         /// It is likely not called more than once in an action.
         /// </summary>
-        private async Task<IPackageReferenceContextInfo> GetInstalledPackageAsync(
+        private async Task<(IPackageReferenceContextInfo TopLevelPackage, IPackageReferenceContextInfo TransitivePackage)> GetInstalledPackageAsync(
```

Comment:
Throughout the code, when we return just top-level packages, we have a method called GetInstalledPackagesAsync, but when we return top-level and transitive packages, we call it GetInstalledAndTransitivePackagesAsync. For consistency, we should consider doing that here now that this method returns both.